### PR TITLE
Add  banner that prompts user to enable restores

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

As part of the cleaning of some Backup screens (see #46162 and #46165), this PR adds a banner that prompts the user to set up server credentials.

### Testing instructions

- Download the PR
- Run Calypso with `yarn start`
- Run Jetpack cloud concurrently with `yarn start-jetpack-cloud-p`
- Pick a self-hosted Jetpack site with Backup, and server credentials not set yet
- Visit `/backup/:site`
- Check that you see the banner in both environments, and that the copy matches the captures below
- Check that clicking the button redirects you to the credentials form
- Fill the credentials form
- Check that you don't see the banner anymore

#### Jetpack cloud
_Desktop_
Screen Shot 2020-10-06 at 11 52 38 AM
_Mobile_
Screen Shot 2020-10-06 at 11 52 21 AM

#### Calypso
_Desktop_
Screen Shot 2020-10-06 at 12 07 42 PM
_Mobile_
Screen Shot 2020-10-06 at 12 07 55 PM

### CODE REVIEW

**Review by elliottprogrammer — COMMENTED** [review]
Works and looks good on Calypso & Jetpack Cloud. Code looks great, 👍   I'm just not sure we have the right copy.  See my comment and let me know if I am mistaken.  Thanks @monsieur-z ! 🙂

File: client/my-sites/backup/main.jsx [hunk 1]
```diff
@@ -185,6 +187,23 @@ class BackupsPage extends Component {
 				{ ! isLoadingBackups && (
 					<div className="backup__main-wrap">
 						<div className="backup__last-backup-status">
+							{ isEnabled( 'jetpack/backup-simplified-screens' ) && doesRewindNeedCredentials && (
+								<Banner
+									className="backup__restore-banner"
+									callToAction={ translate( 'Enable restores' ) }
+									title={ translate( 'Set up your server credentials to get back online quickly' ) }
+									description={ translate(
+										'Add SSH, SFTP or FTP credentials to enable one click site restores.'
```
[the remark this answers is not in the document]
Comment (monsieur-z) [reply client/my-sites/backup/main.jsx:196]: @elliottprogrammer  I should have mentioned: the mockups have been updated with the new copy (see issue description in board).

[the remark this answers is not in the document]
Comment (elliottprogrammer) [reply client/my-sites/backup/main.jsx:196]: Ahh, got it!  Thank you! 👍

**Review by elliottprogrammer — APPROVED** [review]
LGTM!  🙂

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4837755

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4837755).

**Pull request merged**